.eslintrc

```diff
@@ -1,44 +1,40 @@
-/* Galaxy's Javascript linting configuration (Meteor)
- *
- * Documentation on rules can be found at:
- * http://eslint.org/docs/rules/ <- Optionally append the rulename
- *
- * Ensure changes are kept in sync with the base linting configuration.
-*/
-
 {
-  "extends": ".eslintrc-meteor",
-  "env": {
-    /* Allows global vars from the Meteor environment to pass and enables certain rules */
-    "meteor": true
+  "extends": [
+    "eslint:recommended"
+  ],
+  "parser": "babel-eslint",
+  "parserOptions": {
+    "ecmaVersion": 6,
+    "sourceType": "module"
   },
   "rules": {
-
-    /* ---------------- Rules relaxed for Meteor's sake --------------------- */
-
-    /* allows use of var (remove when we are ready to switch to ES6) */
-    "no-var": 0,
-
-    /* allows anonymous functions */
-    "func-names": 0,
-
-    /* allows var = function() delcarations */
-    "func-style": 0,
-
-    /* relaxes max-len to 100 chars per line */
-    "max-len": [2, 100, 2],
-
-    /* relaxes requirement to put all vars at the top of the scope */
-    "vars-on-top": 0,
-
-    /* relaxes dangling commas enforcement */
+    "babel/generator-star-spacing": 0,
+    "babel/new-cap": 1,
+    "babel/array-bracket-spacing": 0,
+    "babel/object-curly-spacing": 0,
+    "babel/object-shorthand": 0,
+    "babel/arrow-parens": 0,
+    "babel/no-await-in-loop": 1,
     "comma-dangle": 0,
-
-    /* allows certain non-constructor functions to start with a capital letter, */
-    "new-cap": [2, { "capIsNewExceptions": [
-        "Match", "Any", "Object", "ObjectIncluding", "OneOf", "Optional", "Where"
-    ]}],
-
-    /* ---------------------------------------------------------------------- */
+    "key-spacing": 0,
+    "no-extra-boolean-cast": 0,
+    "no-undef": 1,
+    "no-unused-vars": 1,
+    "no-console": 1
+  },
+  "env": {
+    "es6": true,
+    "node": true,
+    "browser": true,
+    "commonjs": true
+  },
+  "plugins": [
+    "babel",
+  ],
+  "modules": true,
+  "root": true,
+  "globals": {
+    "param": true,
+    "returns": true
   }
```

Comment:
this may have a correlation to this one particular eslint rule that disables calling out `import` and `export` statements not being the first/top declaration in the file.